source/core/Date.cpp

```diff
@@ -314,12 +314,8 @@ namespace Vireo {
 // #if kVireoOS_emscripten formerly here which was using jsTimestampGetTimeZoneOffset has been deleted; the localtime_r emulation works correctly
```

Comment:
I think we should delete all references of jsTimestampGetTimeZoneOffset from .cpp and especially .js  to make sure we don't accidentally mix and match relying on emscriptened native functionality and JS functionality and deal with errors due to differences.